<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>812</width>
    <height>539</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(192, 97, 203);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="10" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Введите название:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(220, 138, 221);</string>
      </property>
      <property name="text">
       <string>Описание:</string>
      </property>
     </widget>
    </item>
    <item row="10" column="1">
     <widget class="QLineEdit" name="nameLineEdit"/>
    </item>
    <item row="12" column="1">
     <widget class="QLineEdit" name="descrLineEdit"/>
    </item>
    <item row="12" column="5">
     <widget class="QPushButton" name="downloadButton">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Загрузить</string>
      </property>
     </widget>
    </item>
    <item row="12" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Введите описание:</string>
      </property>
     </widget>
    </item>
    <item row="12" column="2">
     <widget class="QPushButton" name="saveButton">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Сохранить</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QPushButton" name="addPhotoButton">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Добавить фото</string>
      </property>
     </widget>
    </item>
    <item row="10" column="2" colspan="4">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Поменять</string>
      </property>
     </widget>
    </item>
    <item row="9" column="2" colspan="4">
     <widget class="QPushButton" name="redButton">
      <property name="text">
       <string>Редактировать</string>
      </property>
     </widget>
    </item>
    <item row="8" column="2" colspan="4">
     <widget class="QPushButton" name="deliteButton">
      <property name="text">
       <string>Удалить</string>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QPushButton" name="addButton">
      <property name="text">
       <string>Добавить</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" rowspan="8">
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>343</width>
         <height>403</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="descrLabel">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(220, 138, 221);</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="0" column="0" rowspan="9">
     <widget class="QTreeView" name="treeView">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(220, 138, 221);</string>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
     </widget>
    </item>
    <item row="0" column="2" rowspan="4" colspan="4">
     <widget class="QLabel" name="photoLabel">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(97, 53, 131);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="2" colspan="4">
     <widget class="QLabel" name="label_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0.0201005 rgba(202, 142, 251, 255), stop:0.894472 rgba(252, 183, 252, 255));</string>
      </property>
      <property name="text">
       <string>Название:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2" colspan="4">
     <widget class="QLabel" name="nameLabel">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>URW Gothic</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0.0201005 rgba(202, 142, 251, 255), stop:0.894472 rgba(252, 183, 252, 255));</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
